### ISSUE TITLE
Meeting with CTM on Digital Accessibility Policy 

### ISSUE THREAD

**Issue by dianamartin** [cityofaustin/atd-data-tech#13539]
# 8/25 Notes
Participants: Sai Kiran, Chia, Karo, Diana, Peggy

His team tried to accessibility efforts in web services team. Started out with a tool, 60 and ended up with 88 accessibility (improvement). The ISO security is down, but will try to get to get measurements for accessibility. 

How to rectify those issues

# Tools
- `Site Improve` - CPIO (give license to each department, each department has an account)
- will run all the pages
- CPIO, Caleb S.  is CPIO Manager, digital services team
  - Peggy asked: Can we have a back up SPOC?
  >We have to ask CPIO
- `Wave` is another tool

# Next Step
- once we identify the tool we want to use to run against our apps
- Whoever is the developer (ex. contrast issue, color)
  - there will be suggestions, background colors
- It will recalculate a new score
- working with a 3rd party, low score, 6 - 1 year (can get an exception that goes to CTM, Kerrica L.)
  - alternative: exception route 
- Working with a tool, department wants to see if things are being resolved (15-30 minutes)
  - Sai Kiran or someone from his team can be assigned to help and see if we're moving in the right direction
- Auditor's Office will appreciate this effort

## Timeline?
- Takes effect on September 1st,  2023
- Spend the next week writing exception documents?
- Grace period?
- Memo will go out September 1st
  - there will be time 
  - can put it in the exception
  - MS Form, PowerApps 
  
2013 Audit
- This was one of the initiatives from the Auditor's Office

TPW
- Should include our PIO, does have external content
- Internal/External applications
- Owner of Accessibility

Trying to commit to (at CTM) all applications will be in compliance with accessibility standards

Ex. Vision Zero (austintexas.gov) and custom Vision Zero custom
- header/footer - CTM team will be responsible for it
- PIO will be responsible for auditing the Drupal site

Ex. Implement a translation
RFP, vendors have to be 100% accessibility
Can't come in with bad accessibility scores, it's been communicated to them already

**Comment by KaroEngstrom** [comment]
### Meeting Notes 8/25/23 

Attendance: Diana, Karo, Sai K.(CTM), Peggy, Chia
- Diana starting a Github issue 

Notes: 
- CPIO - SiteImprove - plan to give a license to each department (one SPOC, one account) 
    - Can this have a backup? Can we use the service account? 
- Once we have this, we can run all the pages 
- Caleb Sanchez is the new supervisor for CPIO digital team 
- Also Weave - CTM not personally using it 
- Auditor had specific tools to look at
- Will SiteImprove also work on PDFs and SharePoint pages. video, multimedia? 
    - Question for CPIO 
- Once we recognize the tools we want to run out items through, it gives you suggestions to modify 
- If the score is very low, you can get an exception with CTM - this comes with a deadline and need to have a specific plan 
- When will we need an exception? Is there any time to get started from the initial launch? - will find out from the communications coming soon
- Sai or someone from his team can offer expert suggestions on the accessibility issues
- Memo will go out Sept 1 - exception if needed is pretty simple; can include multiple application and such; exception applies to the whole department 
- Broken links a huge issue for the entire city, all departments - accessibility issue 
- This sound like something we need to involve our PIO/Jeff in as well 
- Each department owns accessibility
- Goal to get everything in compliance 
- Chia - Drupal site displaying the Vision Zero content? 
    - Footer and header definitely falls to the website 
    - App items need to come from the department creator 
    - Drupal page links to widgets to link to the external page
- PIO needs to be involved; hundreds of story pages in ODP, for instance
- Third party compliance - they need to meet compliance within a certain timeframe 
    - Example: translation for the website - vendor must be 100% accessibility 
    - Microsoft largely accessibly 
    - What about platforms for building and publishing assets? 
        - Socrata - CTM is looking at this 
        - Knack - need to go to them and ask them to be accessible - start the conversation next month on this 
        - Formstack - have not talked to them yet; CTM will be talking to them 
        - Maximo - also an Enterprise application; each manager will be looking into this 
- Executive order to federal department to respond to this in 2020 
- Enterprise apps will be handled by CTM? - Not sure, what we build needs to be accessible
- Wave https://wave.webaim.org/ 
- Modifications to contracts to require accessibility? Written into contracts from now on? - Add to RFP, if you tell central purchasing to include the accessibility piece; if we find discrepancy, we need to reach out and try to negotiate accessibility piece 
- Once we’ve scanned the website, and it’s ok, do we report it somehow? 
    - If you are compliant, there’s a form to submit (can send an email to web services team) 
    - Need to check every update! 
    - SiteImprove model 
- Sept 1 - more information
- Asking each department to own digital accessibility 
- What’s the limit for compliance? Not a Score but a state, assessed by the tool or assessor, depending on the types and levels or errors

**Comment by KaroEngstrom** [comment]
## Links 

🔗  [Policy text](https://cityspace.austintexas.gov/files/assets/workspace/ctm/coa-digital-accessibility-policy-signed.pdf)

🔗  [CTM Digital Services home](https://cityofaustin.sharepoint.com/sites/CTM/TEAMSITES/WEB/SitePages/Services.aspx)

🔗 [ Wave assessment tool](https://wave.webaim.org/)

🔗 [ Web Content Accessibility Guidelines 2.2 (WCAG)](https://www.w3.org/TR/WCAG22/)

🔗  [2023 Digital Accessibility Updates folder](https://drive.google.com/drive/folders/1orWZxWU4nJvjfZ53CLQItAXAz7DvO_1F?usp=drive_link)

**Comment by dianamartin** [comment]
@KaroEngstrom is this okay to close?

**Comment by KaroEngstrom** [comment]
@dianamartin Yes. We have more info and we have access to the SiteImprove tool. Next step would be working out a plan for DTS to do this but that should be its own issue/epic. Closing.
